### ISSUE TITLE
play_from_queue() returns None instead of True

### ISSUE THREAD

**Issue by ghcs27** [SoCo/SoCo#509]
The docstring of `play_from_queue()` states
```
Returns:
    True if the Sonos speaker successfully started playing the track.
    False if the track did not start (this may be because it was not
    requested to start because "start=False")
```
However, it currently returns None if `start=True` and False if `start=False`.

There is no reason for `play_from_queue()` to return a success indicator: If it fails, it will raise an exception.

IMO, we should remove the return value. What do you think?

**Comment by DPH** [comment]
I agree it serves no purpose.

**Comment by ghcs27** [comment]
Same with `pause`, `stop`, `seek`, `next` and `previous`. This is byte size.

**Comment by KennethNielsen** [comment]
@ghcs27 these are errors in the doc strings, not the code, as you have set out to fix in #540. Will close this one.
